### PR TITLE
Peak demand annual savings All Tests Passing

### PR DESCRIPTION
**Pull request by SelvaRC**
adding new variable Peak demand annual savings

### CODE REVIEW

File: openfisca_nsw_safeguard/tests/ESS_PDRS_Estimator/WH1/WH1_PRC_calculation.yaml [hunk 1]
```diff
@@ -56,6 +56,24 @@
         0.73825
       ]
 
+- name: test WH1 peak demand annual savings
+  period: 2024
+  absolute_error_margin: 0.1
+  input:
+    SYS2_peak_demand_savings_capacity:
```
Comment (alisha17) [inline openfisca_nsw_safeguard/tests/ESS_PDRS_Estimator/WH1/WH1_PRC_calculation.yaml:63]: This isn't right. This file is for WH1 but the variable is SYS2.

**Comment by alisha17** [comment]
Are all the tests passing?

File: openfisca_nsw_safeguard/tests/ESS_PDRS_Estimator/WH1/WH1_PRC_calculation.yaml (hunk 1 above)
Comment (SelvaRC) [reply openfisca_nsw_safeguard/tests/ESS_PDRS_Estimator/WH1/WH1_PRC_calculation.yaml]: File has now been updated to reflect correct variable.

**Comment by SelvaRC** [comment]
> Are all the tests passing? @SelvaRC

Yes

**Review by alisha17 — APPROVED** [review]

**Pull request merged**